.github/workflows/tox-pytest.yml

```diff
@@ -33,11 +36,13 @@ jobs:
           conda config --show
           printenv | sort
 
-      # Authentication via credentials json
       - name: Authenticate gcloud
         uses: "google-github-actions/auth@v1"
+        continue-on-error: true
```

Comment:
The idea with having authentication for  `tox-pytest` continue on error is that it would fall back to direct Zenodo data access? Which shouldn't be an issue in this repo I don't think since it's not pulling any of that data -- but just for uniformity?